plugins/node/opentelemetry-instrumentation-mongodb/test/mongodb-v4-v5.metrics.test.ts

```diff
@@ -83,31 +94,31 @@ describe('MongoDBInstrumentation-Metrics', () => {
 
   const HOST = process.env.MONGODB_HOST || DEFAULT_MONGO_HOST;
   const PORT = process.env.MONGODB_PORT || '27017';
-  const DB_NAME = process.env.MONGODB_DB || 'opentelemetry-tests-metrics';
+  const DB_NAME = process.env.MONGODB_DB || 'opentelemetry-tests';
```

Comment:
why change this? :thinking:

Reply:
I had some issues with the pool names, but you're right, I shouldn't have changed it. Reverted